### ISSUE TITLE
Compatibility with next.js v14

### ISSUE THREAD

**Issue by TheMikeyRoss** [jaydenseric/next-graphql-react#7]
I got this issue after updating my next.js to version 14.0.1

```
npm ERR! Could not resolve dependency:
npm ERR! peer next@"12 - 13" from next-graphql-react@15.0.2
npm ERR! node_modules/next-graphql-react
npm ERR!   next-graphql-react@"^15.0.2" from the root project
```

**Comment by jaydenseric** [comment]
Yes I have been working on compatibility with Next.js 14 this week, but got bogged down by having to redo a lot of tests because they annoyingly removed the `next export` command.

**Comment by TheMikeyRoss** [comment]
@jaydenseric no problem brother, take your time. I appreciate your work 👍

I reverted back to next.js v13.5.0 for now

**Comment by jaydenseric** [comment]
This is fixed in [v16.0.0](https://github.com/jaydenseric/next-graphql-react/releases/tag/v16.0.0) 🚀
